bazel/repository_locations.bzl

```diff
@@ -80,13 +80,13 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         # 2. Open https://chromium.googlesource.com/chromium/src/+/refs/tags/<current_version>/DEPS and note <boringssl_revision>.
         # 3. Find a commit in BoringSSL's "master-with-bazel" branch that merges <boringssl_revision>.
```

Comment:
Follow this step, the version you want to use is `62079f7cb4e9a2d5d8a68fe8a4a3f2375dd53585`.

Reply:
Thanks, I updated the version, hope it's ok now.